### PR TITLE
Upgrade to 0.93

### PR DESCRIPTION
**Pull request by hcho3**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)

### CODE REVIEW

**Comment by chyunsu3** [comment]
@conda-forge-admin, please rerender

**Comment by chyunsu3** [comment]
@conda-forge-admin, please rerender

**Comment by chyunsu3** [comment]
@conda-forge-admin, please rerender

**Comment by jakirkham** [comment]
`bld.bat` should be in the `recipe` directory.

**Comment by chyunsu3** [comment]
@jakirkham My bad. Thanks!

**Comment by jakirkham** [comment]
No worries. Looks like you were already fixing it 🙂

**Comment by jakirkham** [comment]
@conda-forge-admin, please re-render.

**Comment by jakirkham** [comment]
Maybe we can disable the static library on Windows? Or do we need that for things downstream of this package?

**Comment by chyunsu3** [comment]
Let me create a patch for removing static libs from the build.

**Comment by chyunsu3** [comment]
@conda-forge-admin, please rerender

File: recipe/bld.bat [hunk 1]
```diff
@@ -0,0 +1,12 @@
+mkdir build
```
Comment (jakirkham) [inline recipe/bld.bat:1]: We may also need this...

```suggestion
set PROTOBUF_USE_DLLS=1
mkdir build	
```

https://github.com/protocolbuffers/protobuf/blob/214c77e1b76e63e512bd675d1c300c80438642b6/cmake/README.md#dlls-vs-static-linking
https://github.com/protocolbuffers/protobuf/issues/2502#issuecomment-412868718

**Comment by jakirkham** [comment]
Looks like Python 3.6 and 3.7 now pass! 😄

Python 3.8 fails, but this appears to be related to a known issue finding DLLs ( https://github.com/conda-forge/python-feedstock/issues/307 ). We've seen this before in `llvmlite` where we added a small patch to include the right search path ( https://github.com/conda-forge/llvmlite-feedstock/pull/44 ), which fixed the issue there. Maybe we can do something similar here?

**Comment by chyunsu3** [comment]
@jakirkham Thanks! Hopefully the new fix will fix the DLL path issue.

File: recipe/0003-py38-windows-dll-path-fix.patch [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
++import os
+ import ctypes
+ 
+ import numpy as np
+@@ -16,6 +17,8 @@ from .libpath import find_lib_path, TreeliteLibraryNotFound
+ def _load_lib():
+     """Load Treelite Library."""
+     lib_path = find_lib_path()
++    if sys.version_info >= (3, 8) and sys.platform == 'win32':
++        os.add_dll_directory(os.path.dirname(lib_path[0]))
```
Comment (jakirkham) [inline recipe/0003-py38-windows-dll-path-fix.patch]:
```suggestion
+        os.add_dll_directory(os.path.join(os.path.normpath(sys.prefix), 'Library', 'bin'))
```

File: recipe/0003-py38-windows-dll-path-fix.patch [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+diff --git a/runtime/python/treelite_runtime/predictor.py b/runtime/python/treelite_runtime/predictor.py
+index 4d15a64..9b21a55 100644
+--- a/runtime/python/treelite_runtime/predictor.py
++++ b/runtime/python/treelite_runtime/predictor.py
+@@ -16,6 +16,8 @@ from .libpath import TreeliteRuntimeLibraryNotFound, find_lib_path
+ def _load_runtime_lib():
+     """Load Treelite runtime"""
+     lib_path = find_lib_path()
++    if sys.version_info >= (3, 8) and sys.platform == 'win32':
++        os.add_dll_directory(os.path.dirname(lib_path[0]))
```
Comment (jakirkham) [inline recipe/0003-py38-windows-dll-path-fix.patch]: May need `import sys` in this module as well.

```suggestion
+        os.add_dll_directory(os.path.join(os.path.normpath(sys.prefix), 'Library', 'bin'))
```

**Review by jakirkham — COMMENTED** [review]
Sounds good. If not, we can do something like this, which should work.

**Comment by chyunsu3** [comment]
@jakirkham It worked! Thanks a lot!

**Comment by jakirkham** [comment]
Great thanks for working on this @hcho3! 😀

**Pull request merged**